openwisp_monitoring/device/tests/test_apps.py

```diff
@@ -37,3 +40,54 @@ def test_device_map_template_registered(self):
                 'monitoring/js/device-map.js',
             ),
         )
+
+
+class TestCheckSignals(TestCase):
```

Comment:
The tests are failing because this class is not inheriting the test mixins which provide helper methods like `_create_device`.

Instead of adding a new test class just for these tests, I think we can move these tests to `TestDeviceMonitoring` in openwisp_monitoring/device/tests/test_models.py